### ISSUE TITLE
Here should be model.train()?

### ISSUE THREAD

**Issue by gongel** [lqtrung1998/mwp_ReFT#18]
https://github.com/lqtrung1998/mwp_ReFT/blob/cd8c0ecb5761b6d204af0a24af9f78ca7a7d621d/train_rl_reft.py#L455

Thx~

**Comment by lqtrung1998** [comment]
Hi @gongel 

We intentionally stop dropout by doing model.eval()

Best Regards,
Trung

**Comment by gongel** [comment]
ok thx
